x-pack/plugins/actions/server/builtin_action_types/servicenow/transformers.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { TransformerArgs } from './types';
+
+export const informationCreated = ({
+  value,
+  date,
+  user,
+  ...rest
+}: TransformerArgs): TransformerArgs => ({
+  value: `${value} (created at ${date} by ${user})`,
```

Comment:
we should make each of these `i18n`. for an example... 
```
export const INFORMATION_CREATED = (ruleId: string, statusCode: number, message: string) =>
  i18n.translate(
    'xpack.actions.builtin.servicenow.informationCreated',
    {
      values: { value, date, user },
      defaultMessage: `${value} (created at ${date} by ${user})`,
    }
  );
```

I bet you could probably do a `switch case` and do them all in one translation